docs/accounting/Accounting.lhs

```diff
@@ -0,0 +1,405 @@
+\documentclass[11pt,a4paper]{article}
+%include polycode.fmt
+
+\newcommand{\lovelace}{\ensuremath{\mathsf{lovelace~}}}
```

Comment:
great!

Reply:
I'll add in all these suggestions at once, otherwise it'll trigger the CI each time... I need to figure out how to avoid that with the new github "apply suggestions", because otherwise it is such a great feature.